src/Umbraco.Web/Editors/MediaController.cs

```diff
@@ -278,6 +281,31 @@ public IEnumerable<MediaItemDisplay> GetByIds([FromUri]int[] ids)
         }
 
         /// <summary>
+        /// Searches media and returns a paged result
+        /// </summary>
+        /// <param name="query"></param>
+        /// <param name="pageNumber"></param>
+        /// <param name="pageSize"></param>
+        /// <param name="searchFrom"></param>
+        /// <returns></returns>
+        [HttpGet]
+        [FilterAllowedOutgoingMedia(typeof(IEnumerable<ContentItemBasic<ContentPropertyBasic, IMedia>>), "Items")]

```

Comment:
@Shazwazza I had two reasons for not using the GetChildren method: 
One - if a start node is set in the GetChildren then it doesn't search the underlying tree - only children having ParentID = startNode - but I guess we can find a way around that?
Second - the Content Picker uses the examine search method for it's search functionality so to keep it consistent I went with the same approach - but as we needed the properties for showing images (we need a list of `ContentItemBasic<ContentPropertyBasic, IMedia>` I thought it didn't belong in the EntityController and therefore I moved it to the MediaController.